types/company.ts

```diff
@@ -1,5 +1,12 @@
-export type CompanyData = {
+export type CompanyListData = {
   id: number;
   name: string;
   companyUrl: string;
 };
```

Comment:
Super nit

```suggestion
export type CompanyListData = CompanyPostData & {
  id: number;
};
```

Reply:
I think we should not do this as the incoming POST data should not be tied to the outgoing response data.